### DISCUSSION TITLE
why photo color is changed when I use function -> DrawImage

### DISCUSSION THREAD

**Discussion by 3egirlsdream** [SixLabors/ImageSharp#2205] [Q&A]
I use `DrawImage` put a photo on a whitepaper, and `Save` it ,its color become more lighter.

**Comment by JimBobSquarePants** [comment]
I dunno. Why don't you provide some information so we can figure it out together?

**Reply by 3egirlsdream** [reply]
````
var img = Image.Load("");
var newImg = new Image<Rgba32>(img.Width, img.Height);
var startPos = new SixLabors.ImageSharp.Point(0, 0);
newImg.Mutate(x => x.DrawImage(img, startPos, 1));
newImg.Save("");
````
- source image
![http://cdn.thankful.top/DSC02154_103709.jpg](http://cdn.thankful.top/DSC02154_103709.jpg)

**Reply by JimBobSquarePants** [reply]
Can you try posting the image again? I'm getting a content length exceeded error.
